core/lib/spree/core/search/base.rb

```diff
@@ -4,6 +4,12 @@ module Spree
   module Core
     module Search
       class Base
+        class InvalidOptions < StandardError
```

Comment:
```suggestion
        class InvalidOptions < ArgumentError
```
Thoughts about being more specific?

Reply:
Nice suggestion, going to change it.